packages/mobile/src/services/verify-phone-api/verify-phone-api.service.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  HttpContentType,
+  HttpMethod,
+  ApiRoutes,
+  VerifyApiRoutes,
+  PhoneVerifyDto,
+} from '@vse-bude/shared';
+import { Http } from '../http/http.service';
+
+type Constructor = {
+  http: Http;
+  apiPrefix: string;
+};
+
+class VerifyPhoneApi {
+  #http: Http;
+
+  #apiPrefix: string;
+
+  constructor({ http, apiPrefix }: Constructor) {
+    this.#http = http;
+    this.#apiPrefix = apiPrefix;
+  }
+
+  sendCodeVerifyPhone(): Promise<boolean> {
```

Comment:
```suggestion
  sendVerificationCode(): Promise<boolean> {
```

Don't we need to specify payload (code itself) as well?

Reply:
no it's for getting the code